src/solvers/td3.ml

```diff
@@ -980,17 +986,39 @@ module WP =
       end
       in
 
-      (* postsolver also populates side_dep and side_infl *)
+      (* postsolver also populates side_dep, side_infl, and dep *)
       let module SideInfl: PostSolver.S with module S = S and module VH = HM =
       struct
         include PostSolver.Unit (S) (HM)
 
         let one_side ~vh ~x ~y ~d =
+          (* Also record side-effects caused by post-solver *)
           HM.replace side_dep y (VS.add x (try HM.find side_dep y with Not_found -> VS.empty));
           HM.replace side_infl x (VS.add y (try HM.find side_infl x with Not_found -> VS.empty));
       end
       in
 
+      let reachable_and_superstable =
+        if incr_verify then
```

Comment:
I'm thinking maybe we should have an additional option for whether to conservatively use the entire `superstable` for this or have this additional `uses`-reachability phase in between.